com.github.robertsanseries.ciano.json

```diff
@@ -0,0 +1,130 @@
+{
+    "app-id": "com.github.robertsanseries.ciano",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.26",
+    "sdk": "org.gnome.Sdk",
+    "command": "com.github.robertsanseries.ciano",
+    "rename-icon": "com.github.robertsanseries.ciano",
+    "rename-appdata-file": "com.github.robertsanseries.ciano.appdata.xml",
+    "build-options": {
+        "env": {
+            "PKG_CONFIG_GOBJECT_INTROSPECTION_1_0_GIRDIR": "/app/share/gir-1.0",
+            "PKG_CONFIG_GOBJECT_INTROSPECTION_1_0_TYPELIBDIR": "/app/lib/girepository-1.0"
+        }
+    },
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--device=all",
+        "--socket=wayland",
+        "--filesystem=home",
+        "--talk-name=ca.desrt.dconf",
+        "--filesystem=xdg-run/dconf",
+        "--filesystem=~/.config/dconf:ro",
+        "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+    ],
+    "modules": [
+        {
+            "name": "libgee",
+            "build-options": {
+                "env": {
+                    "PKG_CONFIG_GOBJECT_INTROSPECTION_1_0_GIRDIR": "/app/share/gir-1.0",
+                    "PKG_CONFIG_GOBJECT_INTROSPECTION_1_0_TYPELIBDIR": "/app/lib/girepository-1.0"
+                }
+            },
+            "sources": [
+                {
+                    "type":"archive",
+                    "url": "https://github.com/GNOME/libgee/archive/0.20.0.tar.gz",
+                    "sha256": "42fe6d651910cb8b720167f71c5255a1b7b1afc82fecd3f31e61f9602b3b1335"
+                }
+            ]
+        },
+        {
+            "name": "granite",
+            "buildsystem": "cmake-ninja",
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=Release",
+                "-DCMAKE_INSTALL_LIBDIR=/app/lib"
+            ],
+            "cleanup": [
+                "/bin",
+                "/include",
+                "/share/vala",
+                "/share/gir-1.0",
+                "/share/applications",
+                "/lib/pkgconfig",
+                "/lib/girepository-1.0",
+                "/lib/*.la"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://launchpad.net/granite/0.4/0.4.1/+download/granite-0.4.1.tar.xz",
+                    "sha256": "8bb411523e6cd5685c928dbfe08893892189cedf9fad5f537aa76182032cf09c"
+                }
+            ]
+        },
+        {
+            "name": "ffmpeg",
```

Comment:
ffmpeg uses on flathub should specify the codecs and options they want in order to reduce the potential surface area to attack and overall size. Have a look at the [many, many other configurations available](https://github.com/search?q=org%3Aflathub+ffmpeg&type=Code) ;-)